### PR TITLE
fix: add Cao Tan Duc

### PR DESCRIPTION
**Pull request by caotanduc**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/907"></a>

### CODE REVIEW

**Review by avneesh0612 — APPROVED** [review]
LGTM 🔥🚀

File: public/data/caotanduc.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "Facebook"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "University",
+      "date": "September 2020",
+      "icon": "book",
+      "color": "grey",
+      "description": "Start studying at Ho Chi Minh University of Science - Vietnam"
```
Comment (krishguptadev) [inline public/data/caotanduc.json:24]: Tense error 😅 

```suggestion
      "description": "Started studying at Ho Chi Minh University of Science - Vietnam"
```

Comment (caotanduc) [reply public/data/caotanduc.json]: 👍 Thanks for your help. I am not good at English so I used Google translate for this sentence, but it doesn't work well with my Mother tongue.

**Review by krishguptadev — APPROVED** [review]

**Review by syedsohan — CHANGES_REQUESTED** [review]
Change the PR title to - `fix: add Cao Tan Duc`

File: public/data/caotanduc.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/caotanduc",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Facebook",
+      "url": "https://facebook.com/tan.duc.92317/",
+      "icon": "Facebook"
```
Comment (syedsohan) [inline public/data/caotanduc.json:15]:
```suggestion
      "icon": "facebook"
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/caotanduc.json (hunk 2 above)
Comment (caotanduc) [reply public/data/caotanduc.json]: Thanks for your help, I appreciate it!

**Review by syedsohan — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Pull request merged**